### PR TITLE
[FrameworkBundle][Routing] Added doc for new "template", "redirect" shortcuts

### PR DESCRIPTION
**Pull request by HeahDude**
Added documentation for symfony/symfony#30501 and symfony/symfony#35653

### CODE REVIEW

File: routing.rst [hunk 1]
```diff
@@ -1333,68 +1333,71 @@ Use the ``RedirectController`` to redirect to other routes and URLs:
...
+                permanent="true">
+            </url-redirect-route>
         </routes>
 
     .. code-block:: php
 
         // config/routes.php
         use App\Controller\DefaultController;
-        use Symfony\Bundle\FrameworkBundle\Controller\RedirectController;
         use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
```
Comment (B-Galati) [inline routing.rst]: Should it used the RoutingConfigurator of the bundle or the component ?

Comment (HeahDude) [reply routing.rst]: The bundle, good catch! thx :)

**Review by nicolas-grekas — APPROVED** [review]

**Comment by OskarStark** [comment]
@HeahDude If I am not wrong, the corresponding code is merged, could you please rebase and remove (If I am right) the `Waiting Code Merge` label? Thanks

**Comment by wouterj** [comment]
As the feature is reverted from 5.1 and the related PRs are closed, let's close this PR as well. The branch of this PR remains on GitHub and can be used when there is a consensus on https://github.com/symfony/symfony/issues/28806

**Pull request closed without merging**